### ISSUE TITLE
item set update removes all team association

### ISSUE THREAD

**Issue by mquantin** [UIUCLibrary/teams#126]
Creating an item set with teams association is working fine.
Updating that item set (any update, even no changes) removes all teams association. 
The update is applied.
Then the teams is removed.

As Global Administrator: It happens silently. Message is "Item set successfully updated". No log is produced
As User: I get 
```
Omeka\Api\Exception\PermissionDeniedException: Permission denied. Resource "Omeka\Entity\ItemSet:xxx" is not part of your current team, yyyy.
If you feel this is an error, try changing teams or talk to the administrator.
Action: read
```
(with xxx the itemSet id and yyy the team name.)
->This log only concerns the impossibility to display the itemSet after the team have been removed (and after the other edit applied). It's logical. 

So no log is produced for undesired "team removal" event.

It happens to any newly created item set and old ones.

It's a new bug we didn't notice before. 
Either because it did not exist (what could have happend?  Teams module have not been updated)
Either because we did not update our item set values. But I don't think so. 

Very strange...
Recently, module _Numeric Data Types_ and module _File Side Load_ have been installed. I tried to deactivate them. Nothing changed.

I can provide access to the omeka instance for tests if needed...

**Comment by alexdryden** [comment]
Hi @mquantin thanks for the report! This was a bug that was recently introduced when I changed a form parameter and forgot to update in all of the controllers that use the form. It should be fixed in the develop branch (see #125 )and will be part of the v4.x release, which I'm hoping to get out tomorrow or early next week. 

v4 release is taking a bit longer because after I discovered this bug I decided to write some additional functional tests to help catch bugs like this. I'm about 30% through writing those tests.
